dotCMS/src/main/webapp/html/js/dotcms/dijit/AddToBundleDialog.js

```diff
@@ -45,7 +45,6 @@ dojo.declare("dotcms.dijit.AddToBundleDialog", null, {
         dojo.connect(dia, "onDownloadEnd", function () {
             var lastSelectedBundle = JSON.parse(sessionStorage.getItem("lastSelectedBundle"));
             if (lastSelectedBundle && lastSelectedBundle.name) {
-                dojo.byId("bundleName").value = lastSelectedBundle.name;
```

Comment:
are you sure this function is not used by another functionality? I think it's safer to validate if the field exists instead or removing the line

Reply:
done https://github.com/dotCMS/core/pull/17920/commits/3060b57d02d3a1f03dfec3431efa923e908205d6